### PR TITLE
chart release 2.7.9

### PR DESCRIPTION
**Pull request by ganga1980**
chart release 2.7.9

### CODE REVIEW

File: repo/index.yaml [hunk 1]
```diff
@@ -93,6 +93,49 @@ entries:
...
+      name: ganga1980
+    name: azuremonitor-containers
+    sources:
+    - https://github.com/microsoft/Docker-Provider/tree/ci_prod
+    urls:
+    - azuremonitor-containers/azuremonitor-containers-2.7.9.tgz
+    version: 2.7.9
   - apiVersion: v1
     appVersion: 7.0.0-1
     created: "2020-11-05T08:19:17.8990924-08:00"
```
Comment (vishiy) [inline repo/index.yaml:141]: Please change the date to today's date. You can leave the time as is as it doesn't matter. Date shows up in artifact hub as well as RSS for each chart version.
image

Comment (ganga1980) [reply repo/index.yaml:141]: > Please change the date to today's date. You can leave the time as is as it doesn't matter. Date shows up in artifact hub as well as RSS for each chart version.
> image

> your referring old previous chart release date. for 2.7.9 is  created: "2020-11-12T13:55:43.6867268-08:00"

File: repo/index.yaml [hunk 2]
```diff
@@ -93,6 +93,49 @@ entries:
...
+    - kuberneteshealth
+    kubeVersion: ^1.10.0-0
+    maintainers:
+    - email: visnara@microsoft.com
+      name: vishiy
+    - email: gangams@microsoft.com
+      name: ganga1980
+    name: azuremonitor-containers
+    sources:
+    - https://github.com/microsoft/Docker-Provider/tree/ci_prod
```
Comment (vishiy) [inline repo/index.yaml:135]: May be add 'charts/azuremonitor-containers' to point them to this chart directly ?

Comment (ganga1980) [reply repo/index.yaml]: make sense. done.

File: repo/index.yaml (hunk 1 above)
Comment (vishiy) [reply repo/index.yaml:141]: i see date created: "2020-11-05T08:19:17.8990924-08:00". I also see reduntant things - see lines 139,140,141
- apiVersion: v1
    appVersion: 7.0.0-1	    appVersion: 7.0.0-1
    created: "2020-11-05T08:19:17.8990924-08:00"	    created: "2020-11-05T08:19:17.8990924-08:00"

Comment (ganga1980) [reply repo/index.yaml:141]: > i see date created: "2020-11-05T08:19:17.8990924-08:00". I also see reduntant things - see lines 139,140,141
> 
> * apiVersion: v1
>   appVersion: 7.0.0-1
>   created: "2020-11-05T08:19:17.8990924-08:00"

>  line 139,140 and 141 for 2.7.8 release. If you see the full file, you see the created date  top for 2.7.9 on the top at line 98.

2.7.9 
https://github.com/microsoft/charts/blob/842825203202fd82fed3188fc23b536f983b96d6/repo/index.yaml#L98

2.7.8
https://github.com/microsoft/charts/blob/842825203202fd82fed3188fc23b536f983b96d6/repo/index.yaml#L141

**Review by vishiy — APPROVED** [review]

**Pull request merged**